benchmark/CMakeLists.txt

```diff
@@ -15,44 +15,83 @@ include(CTest)
 add_definitions(-D_CHAR16T)
 
 #-------------------------------------------------------------------------------------------
-# Include directories 
+# Include directories
 #-------------------------------------------------------------------------------------------
 include_directories(source)
 include_directories(../test/source)
 
 #-------------------------------------------------------------------------------------------
 # Compiler Flags
 #-------------------------------------------------------------------------------------------
-set(CMAKE_CXX_STANDARD 11)
-
-if (CMAKE_CXX_COMPILER_ID MATCHES "Clang")
-	SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+if(CMAKE_CXX_COMPILER_ID MATCHES "Clang" AND NOT APPLE) # Apple clang uses different versioning.
```

Comment:
Is it possible to have this code in a single file and reference it in the various cmakelist.txt files?  That way we don't have to repeat ourselves and make future changes in multiple places.